usl_pipeline/usl_pipeline/chunkers/elevation_chunkers.py

```diff
@@ -0,0 +1,65 @@
+from osgeo import gdal
+
+from usl_pipeline.readers import elevation_readers
+from usl_pipeline.shared import suppliers
+
+
+class GeoTiffChunker:
```

Comment:
Do you think we need a class here, or could we get away with a top-level function that's 
```python
def split_into_chunks(
    chunk_file_path_generator: suppliers.ChunkFilePathGenerator,
    elevation_file_path: str,
    chunk_size: int)
```

Reply:
Done.